Hashids/Hashids.swift

```diff
@@ -283,84 +256,71 @@ public class Hashids_<T: protocol<Equatable, UnsignedIntegerType>> : HashidsGene
         return Int(trunc(value))
     }
     
-    private func _saltify(inout salt:[Char], _ lottery:Char, _ alphabet:[Char]) -> (Range<Int>, Range<Int>)
-    {
-        salt.append(lottery);
-        salt = salt + self.salt;
-        salt = salt + alphabet;
-        let lsaltARange = (self.salt.count + 1)..<salt.count;
-        let lsaltRange = 0..<alphabet.count;
-        return (lsaltARange, lsaltRange);
+    private func _saltify(inout salt: [Char], _ lottery: Char, _ alphabet: [Char]) -> (Range<Int>, Range<Int>) {
+        salt.append(lottery)
+        salt = salt + self.salt
+        salt = salt + alphabet
+        let lsaltARange = (self.salt.count + 1)..<salt.count
+        let lsaltRange = 0..<alphabet.count
+        return (lsaltARange, lsaltRange)
     }
    
 }
 
 // MARK: Internal functions
 
-internal func contains<T:CollectionType where T.Generator.Element:Equatable>(a:T, e:T.Generator.Element) -> Bool
-{
+func contains<T: CollectionType where T.Generator.Element: Equatable>(a: T, e: T.Generator.Element) -> Bool {
     return a.indexOf(e) != nil
 }
 
-internal func transform<T:CollectionType where T.Generator.Element:Equatable>(a: T, _ b: T, cmpr: (inout Array<T.Generator.Element>, T, T, T.Generator.Element ) -> Void ) -> [T.Generator.Element]
-{
-    typealias U = T.Generator.Element;
-    var c = [U]();
-    for i in a
-    {
-        cmpr(&c, a, b, i);
+func transform<T: CollectionType where T.Generator.Element: Equatable>(a: T, _ b: T, cmpr: (inout Array<T.Generator.Element>, T, T, T.Generator.Element ) -> Void ) -> [T.Generator.Element] {
+    typealias U = T.Generator.Element
+    var c = [U]()
```

Comment:
i c u r m t

Also probably worth re-evaluating some of these value names. Maybe they're okay but I just kind of glazed over when I looked at this.